test/helpers/describe_installer.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+const _ = require('lodash')
+const fs = require('fs-extra')
+const path = require('path')
+const temp = require('temp')
+
+const installer = require('../..')
+
+module.exports = function describeInstaller (description, installerOptions, itDescription, itFunc) {
+  describe(description, test => {
+    const outputDir = installerOptions.dest || module.exports.tempOutputDir()
+
+    before(done => {
+      const options = module.exports.testInstallerOptions(outputDir, installerOptions)
+
+      installer(options, done)
+    })
+
+    it(itDescription, done => {
+      itFunc(outputDir, done)
+    })
+
+    module.exports.cleanupOutputDir(outputDir)
+  })
+}
+
+module.exports.cleanupOutputDir = function cleanupOutputDir (outputDir) {
+  after(done => {
+    fs.remove(outputDir, done)
+  })
+}
+
+module.exports.tempOutputDir = function tempOutputDir () {
+  return temp.path({prefix: 'electron-installer-debian-'})
```

Comment:
Since we're keeping the `tempOutputDir` function instead of replacing all mentions of it with `temp.path`, can we also add a debugging option similar to the `installerOptions.dest`? Maybe do:
```JS
module.exports.tempOutputDir = function tempOutputDir (customDir) {
  return customDir || temp.path({prefix: 'electron-installer-debian-'})
)
```

Reply:
Why not.

Reply:
Well, shouldn't it be:

```js
return customDir ? path.join(os.tmpdir(), customDir) : temp.path({prefix: 'electron-installer-debian-'})
```

Seems a bit weird to have some arbitrary directory not in the temp folder be returned from `tempOutputDir()`.

Reply:
I'm ok your approach. I suggested the previous way because it's more flexible and it's how it was done before. I do agree with you that it's better to keep all these temp directories under the `os.tmpdir` as long as we can specify a name to differentiate it from the rest, we're good